src/function/scalar/system/aggregate_export.cpp

```diff
@@ -179,7 +179,8 @@ static unique_ptr<FunctionData> BindAggregateState(ClientContext &context, Scala
 		throw BinderException("Can only FINALIZE aggregate state, not %s", arg_return_type.ToString());
 	}
 	// combine
-	if (arguments.size() == 2 && arguments[0]->return_type != arguments[1]->return_type) {
+	if (arguments.size() == 2 && arguments[0]->return_type != arguments[1]->return_type &&
```

Comment:
Maybe I am missing something here, but isn't the test really that the two arguments be compatible, not that the second one be a `BLOB`?

Reply:
This seems okay to me, the equality check only triggers if the second argument is NOT a BLOB because if it is a BLOB we can't check